test/bin/eshost.js

```diff
@@ -26,6 +26,8 @@ if (isWindows) {
   });
 }
 
+// To avoid messing with a real configuration, use a test-local config
+Config.defaultConfigPath = path.resolve('./test/config/.eshost-config.json');
```

Comment:
Oh, yes, I think this is unnecessary now. 

> Then you don't need to pass it in.

I determined that passing it in explicitly acts as a reasonable test of `--config` option :)